### PR TITLE
ACE3 v3.14.1 New Settings (Default Values)

### LINKED ISSUE #61: Update ACE3 from v3.13.6 to v3.14.1
ACE3 v3.14.0 has re-introduced a number of new features and settings akin to a mini-overhaul. Some of those features have the potential to change the game experience quite noticeably. In a first step we will have to define the new default settings to retain the status quo of our current gaming experience as much as possible.

### PR DESCRIPTION
**Pull request by JamesTheClarke**
- overview of default new settings for the upcoming change to ACE3 v3.14.1
- fixes #61 

Settings are still subject to change and WIP.

## Changelog New ACE3 Settings / Features

For easy reference (so I can copy-paste the settings directly into the cba_settings.sqf file) I'll summarise my preferred implementation. These are mostly based on @freghar 's recommendations below.

### ACE Advanced Vehicle Damage

``` sqf
// ACE Advanced Vehicle Damage
force force ace_vehicle_damage_enableCarDamage = false;
force force ace_vehicle_damage_enabled = false;
force force ace_vehicle_damage_removeAmmoDuringCookoff = true;
```

- all settings set to `force force` for consistent experience
- `ace_vehicle_damage_enabled` set to `false` as it requires further testing #58 

---

### ACE Common

``` sqf
// ACE Common
ace_common_epilepsyFriendlyMode = false;
ace_common_progressBarInfo = 2;
```

- Default settings, no adjustments required

---

### ACE Cook off

``` sqf
// ACE Cook off
force force ace_cookoff_enableFire = true;
```

- set to `force force` for consistent experience

---

### ACE Dragging

``` sqf
// ACE Dragging
ace_dragging_dragAndFire = false;
```

- set to `false` - players can enable it if they feel comfortable with the feature but it should be disabled by default as a "weapon safety mode" while dragging.

---

### ACE Field Rations

``` sqf
// ACE Field Rations
force acex_field_rations_affectAdvancedFatigue = true;
force acex_field_rations_enabled = false;
acex_field_rations_hudShowLevel = 0;
acex_field_rations_hudTransparency = -1;
acex_field_rations_hudType = 0;
force acex_field_rations_hungerSatiated = 1;
force acex_field_rations_terrainObjectActions = true;
force acex_field_rations_thirstQuenched = 1;
force acex_field_rations_timeWithoutFood = 2;
force acex_field_rations_timeWithoutWater = 2;
force acex_field_rations_waterSourceActions = 2;
```

- ACE3 changed the category name from `// ACEX Field Rations` to `// ACE Field Rations`

---

### ACE Fire

``` sqf
// ACE Fire
force force ace_fire_dropWeapon = 1;
force force ace_fire_enabled = false;
force force ace_fire_enableFlare = false;
force force ace_fire_enableScreams = true;
```

- all settings set to `force force` for consistent experience
- set `ace_fire_enabled` to `false` as it requires further testing #59

---

### ACE Fortify

``` sqf
// ACE Fortify
force ace_fortify_markObjectsOnMap = 1;
force force ace_fortify_timeCostCoefficient = 1;
force force ace_fortify_timeMin = 1.5;
acex_fortify_settingHint = 1;
```

- ACE3 changed the category name from `// ACEX Fortify` to `// ACE Fortify`
- all settings except `acex_fortify_settingHint` and `ace_fortify_markObjectsOnMap` set to `force force` for consistent experience
- set `acex_fortify_settingHint` to `1` - thus fortify budget-change hints only show if you have a fortify tool, non ENGs don't need constant hint pop-ups.

---

### ACE Headless

``` sqf
// ACE Headless
force force acex_headless_delay = 15;
force force acex_headless_enabled = false;
force force acex_headless_endMission = 0;
force force acex_headless_log = false;
force force acex_headless_transferLoadout = 0;
```

- ACE3 changed the category name from `// ACEX Headless` to `// ACE Headless`

---

### ACE Interaction

``` sqf
// ACE Interaction
force ace_interaction_enableGroupRenaming = true;
force force ace_interaction_interactWithTerrainObjects = false;
```

- all settings set to `force force` for consistent experience
- set `ace_interaction_enableGroupRenaming` to `force` - I'm ok to add this temporarily to the latest ACE3 update. However, even with only leaders being able to access I still see more potential for people messing things up / being trolly rather than it solving an actual issue, since in the vast majority of cases Zeus can fix any element name problems on the spot (acts of arma excluded). If this causes people to mess about with the feature during operations I'll call for it to be disabled again.

---

### ACE Interaction Menu (Self) - More

``` sqf
// ACE Interaction Menu (Self) - More
ace_interact_menu_more__ACE_CheckAirTemperature = false;
ace_interact_menu_more__ace_csw_deploy = false;
ace_interact_menu_more__ACE_Equipment = false;
ace_interact_menu_more__ACE_Explosives = false;
ace_interact_menu_more__ace_field_rations = false;
ace_interact_menu_more__ace_fortify = false;
ace_interact_menu_more__ace_gestures = false;
ace_interact_menu_more__ace_intelitems = false;
ace_interact_menu_more__ACE_MapFlashlight = false;
ace_interact_menu_more__ACE_MapGpsHide = false;
ace_interact_menu_more__ACE_MapGpsShow = false;
ace_interact_menu_more__ACE_MapTools = false;
ace_interact_menu_more__ACE_Medical = false;
ace_interact_menu_more__ACE_Medical_Menu = false;
ace_interact_menu_more__ACE_MoveRallypoint = false;
ace_interact_menu_more__ACE_RepackMagazines = false;
ace_interact_menu_more__ace_sandbag_place = false;
ace_interact_menu_more__ACE_StartSurrenderingSelf = false;
ace_interact_menu_more__ACE_StopEscortingSelf = false;
ace_interact_menu_more__ACE_StopSurrenderingSelf = false;
ace_interact_menu_more__ACE_Tags = false;
ace_interact_menu_more__ACE_TeamManagement = false;
ace_interact_menu_more__ace_zeus_create = false;
ace_interact_menu_more__ace_zeus_delete = false;
ace_interact_menu_more__acex_sitting_Stand = false;
ace_interact_menu_more__ACRE_Interact = false;
ace_interact_menu_more__acre_sys_zeus_playerEars = false;
ace_interact_menu_more__acre_sys_zeus_remoteEars = false;
```

- Default settings, no adjustments required

---

### ACE Interaction Menu (Self) - Move to Root

``` sqf
// ACE Interaction Menu (Self) - Move to Root
ace_interact_menu_moveToRoot__ACE_Equipment__a3aa_chestpack_onBack = false;
ace_interact_menu_moveToRoot__ACE_Equipment__a3aa_chestpack_onChest = false;
ace_interact_menu_moveToRoot__ACE_Equipment__a3aa_chestpack_swapPacks = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_atragmx_open = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_attach_Attach = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_attach_Detach = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ACE_CheckDogtags = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ACE_Chemlights = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_dagr_menu = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_dagr_menu__ace_dagr_toggle = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_goggles_wipeGlasses = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_gunbag_actions = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_gunbag_actions__ace_gunbag_status = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_gunbag_actions__ace_gunbag_weaponOff = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_gunbag_actions__ace_gunbag_weaponSwap = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_gunbag_actions__ace_gunbag_weaponTo = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_huntir_open = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_interaction_weaponAttachments = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_kestrel4500_open = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_kestrel4500_open__ace_kestrel4500_hide = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_kestrel4500_open__ace_kestrel4500_show = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_microdagr_configure = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_microdagr_configure__ace_microdagr_close = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_microdagr_configure__ace_microdagr_show = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_minedetector_metalDetector = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_minedetector_metalDetector__ace_minedetector_activate = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_minedetector_metalDetector__ace_minedetector_connectHeadphones = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_minedetector_metalDetector__ace_minedetector_deactivate = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_minedetector_metalDetector__ace_minedetector_disconnectHeadphones = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_mk6mortar_rangetable = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_overheating_CheckTemperature = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_overheating_CheckTemperatureSpareBarrels = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_overheating_CoolWeaponWithItem = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_overheating_SwapBarrel = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_overheating_UnJam = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ACE_PutInEarplugs = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_rangecard_open = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_rangecard_open__ace_rangecard_makeCopy = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_rangecard_open__ace_rangecard_openCopy = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_reload_checkAmmo = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ACE_RemoveEarplugs = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_scopes_adjustZero = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_scopes_resetZero = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_spottingscope_place = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ACE_TacticalLadders = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__ace_trenches_digEnvelopeBig = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__ace_trenches_digEnvelopeSmall = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__grad_trenches_digEnvelopeGiant = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__grad_trenches_digEnvelopeLongNameEmplacment = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__grad_trenches_digEnvelopeShort = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_trenches__grad_trenches_digEnvelopeVehicle = false;
ace_interact_menu_moveToRoot__ACE_Equipment__ace_tripod_place = false;
ace_interact_menu_moveToRoot__ACE_Equipment__acre_sys_gsa_placeSpike = false;
ace_interact_menu_moveToRoot__ACE_Equipment__acre_sys_gsa_placeSpikeMast = false;
ace_interact_menu_moveToRoot__ACE_Explosives__ACE_Cellphone = false;
ace_interact_menu_moveToRoot__ACE_Explosives__ACE_Place = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Advance = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_CeaseFire = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Cover = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Engage = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Follow = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Forward = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Freeze = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Go = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Hold = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Point = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Regroup = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Stop = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Up = false;
ace_interact_menu_moveToRoot__ace_gestures__ace_gestures_Warning = false;
ace_interact_menu_moveToRoot__ACE_MapTools__ACE_MapToolsAlignCompass = false;
ace_interact_menu_moveToRoot__ACE_MapTools__ACE_MapToolsAlignNorth = false;
ace_interact_menu_moveToRoot__ACE_MapTools__ACE_MapToolsHide = false;
ace_interact_menu_moveToRoot__ACE_MapTools__ACE_MapToolsShowNormal = false;
ace_interact_menu_moveToRoot__ACE_MapTools__ACE_MapToolsShowSmall = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_ArmLeft = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_ArmRight = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_Head = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_LegLeft = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_LegRight = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_Torso = false;
ace_interact_menu_moveToRoot__ACE_Medical__ACE_Torso__TriageCard = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_BecomeLeader = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_JoinTeamBlue = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_JoinTeamGreen = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_JoinTeamRed = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_JoinTeamYellow = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_LeaveGroup = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_LeaveTeam = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__ACE_RenameGroup = false;
ace_interact_menu_moveToRoot__ACE_TeamManagement__diwako_dui_buddy_buddy_action_team_remove = false;
```

- Default settings, no adjustments required

---

### ACE Logistics

``` sqf
// ACE Logistics
force force ace_cargo_enableRename = false;
ace_cargo_openAfterUnload = 0;
force force ace_repair_fullRepairRequiredItems = ["ace_repair_anyToolKit"];
force force ace_repair_locationsBoostTraining = true;
force force ace_repair_miscRepairRequiredItems = ["ace_repair_anyToolKit"];
```

- all settings except `ace_cargo_openAfterUnload` set to `force force` for consistent experience
- set `ace_cargo_enableRename` to `false` - custom cargo names will be set in the composition and will not need to be reset during the mission
- set `ace_repair_locationsBoostTraining ` to `true` - levels up the existing repair skill by 1 level if near to designated ACE3 repair facility, equivalent to our existing setting for `ace_medical_treatment_locationsBoostTraining`

---

### ACE Magazine Repack

``` sqf
// ACE Magazine Repack
ace_magazinerepack_repackLoadedMagazines = true;
```

- Default settings, no adjustments required

---

### ACE Medical

``` sqf
// ACE Medical
force force ace_medical_deathChance = 1;
force force ace_medical_enableVehicleCrashes = true;
force force ace_medical_feedback_enableHUDIndicators = true;
ace_medical_gui_bloodLossColor_0 = [1,1,1,1];
ace_medical_gui_bloodLossColor_1 = [1,0.95,0.64,1];
ace_medical_gui_bloodLossColor_2 = [1,0.87,0.46,1];
ace_medical_gui_bloodLossColor_3 = [1,0.8,0.33,1];
ace_medical_gui_bloodLossColor_4 = [1,0.72,0.24,1];
ace_medical_gui_bloodLossColor_5 = [1,0.63,0.15,1];
ace_medical_gui_bloodLossColor_6 = [1,0.54,0.08,1];
ace_medical_gui_bloodLossColor_7 = [1,0.43,0.02,1];
ace_medical_gui_bloodLossColor_8 = [1,0.3,0,1];
ace_medical_gui_bloodLossColor_9 = [1,0,0,1];
ace_medical_gui_damageColor_0 = [1,1,1,1];
ace_medical_gui_damageColor_1 = [0.75,0.95,1,1];
ace_medical_gui_damageColor_2 = [0.62,0.86,1,1];
ace_medical_gui_damageColor_3 = [0.54,0.77,1,1];
ace_medical_gui_damageColor_4 = [0.48,0.67,1,1];
ace_medical_gui_damageColor_5 = [0.42,0.57,1,1];
ace_medical_gui_damageColor_6 = [0.37,0.47,1,1];
ace_medical_gui_damageColor_7 = [0.31,0.36,1,1];
ace_medical_gui_damageColor_8 = [0.22,0.23,1,1];
ace_medical_gui_damageColor_9 = [0,0,1,1];
force force ace_medical_treatment_advancedDiagnose = 2;
force force ace_medical_treatment_clearTrauma = 0;
force force ace_medical_treatment_cprSuccessChanceMax = 0.75;
force force ace_medical_treatment_cprSuccessChanceMin = 0.75;
```

- set `ace_medical_deathChance` to `force force` for consistent experience
- set `ace_medical_enableVehicleCrashes` to `force force` for consistent experience
- set `ace_medical_feedback_enableHUDIndicators` to `force force` for consistent experience
- set `ace_medical_treatment_advancedDiagnose` to `2` - allow players reliably diagnosing death / cardiac arrest via the ACE medical system
- set `ace_medical_treatment_clearTrauma` to `0` and `force force` to retain current behavior
- set `ace_medical_treatment_cprSuccessChanceMax` to `0.75` and `force force` to retain current behavior
- set `ace_medical_treatment_cprSuccessChanceMin` to `0.75` and `force force` to retain current behavior

---

### ACE Name Tags

``` sqf
// ACE Name Tags
force force ace_nametags_ambientBrightnessAffectViewDist = 1;
```

- set to `force force` for consistent experience

---

### ACE Overheating

``` sqf
// ACE Overheating
force force ace_overheating_cookoffCoef = 0;
force force ace_overheating_coolingCoef = 1;
force force ace_overheating_heatCoef = 1;
force force ace_overheating_jamChanceCoef = 1;
force force ace_overheating_overheatingRateOfFire = true;
ace_overheating_particleEffectsAndDispersionDistance = 3000;
force force ace_overheating_suppressorCoef = 1;
force force ace_overheating_unJamOnSwapBarrel = false;
```

- all settings except `ace_overheating_particleEffectsAndDispersionDistance` set to `force force` for consistent experience
- set `ace_overheating_cookoffCoef ` to `0` to retain current behavior, requires further testing #60 

---

### ACE Sitting

``` sqf
// ACE Sitting
force force acex_sitting_enable = true;
```

- Default settings, no adjustments required

---

### ACE Uncategorized

``` sqf
// ACE Uncategorized
force ace_parachute_failureChance = 0;
force force ace_parachute_hideAltimeter = true;
```

- Default settings, no adjustments required

---

### ACE User Interface

``` sqf
// ACE User Interface
force force ace_ui_enableSpeedIndicator = true;
```

- set to `force force` for consistent experience

---

### ACE View Distance Limiter

``` sqf
// ACE View Distance Limiter
ace_viewdistance_enabled = true;
```

- Default settings, no adjustments required

---

### ACE View Restriction

``` sqf
// ACE View Restriction
force force acex_viewrestriction_mode = 0;
force acex_viewrestriction_modeSelectiveAir = 0;
force acex_viewrestriction_modeSelectiveFoot = 0;
force acex_viewrestriction_modeSelectiveLand = 0;
force acex_viewrestriction_modeSelectiveSea = 0;
acex_viewrestriction_preserveView = false;
```

- Default settings, no adjustments required

---

### ACE Volume

``` sqf
// ACE Volume
acex_volume_enabled = false;
acex_volume_fadeDelay = 1;
acex_volume_lowerInVehicles = true;
acex_volume_reduction = 5;
acex_volume_remindIfLowered = false;
acex_volume_showNotification = true;
```

- Default settings, no adjustments required

### CODE REVIEW

**Comment by freghar** [comment]
### Settings to change to retain current ACE behavior

```
force ace_vehicle_damage_enableCarDamage = false;
force ace_vehicle_damage_enabled = true;
force ace_vehicle_damage_removeAmmoDuringCookoff = true;
```
* set all of these to `force force`, they should probably not differ per mission
* set `ace_vehicle_damage_enabled` to `false` to retain current behavior / compatibility

This feature is interesting, but I'd say we should experiment with it first; it does some AI changes too (`allowCrewInImmobile`).

***
```
force ace_cookoff_enableFire = true;
```
* set to `force force` per other settings in the same category, doesn't make sense to be mission-customizable

***
```
force ace_fire_dropWeapon = 1;
force ace_fire_enabled = true;
force ace_fire_enableFlare = false;
ace_fire_enableScreams = true;
```
* set `ace_fire_enableFlare` to `force force` for consistent experience across missions
* consider `force ace_fire_enabled = false;` to retain current behavior
* consider `force force ace_fire_enableScreams = true;` for consistent experience

I'm unsure if the "fire" from cookoff would constitute ACE Fire and make AI go crazy (into weapon-dropping animation) when they disembark, especially in combination with LAMBS. Hence the suggestion to disable it by default, or at least `force ace_fire_dropWeapon = 0;`.

***
```
force ace_interaction_enableGroupRenaming = true;
```
* consider setting this to `force force`, doesn't make sense to be mission-customizable - either we trust our players to not abuse it, or we don't

***
```
ace_cargo_enableRename = true;
```
* consider `force force`, same reason as above

***
```
force ace_medical_deathChance = 1;
force ace_medical_enableVehicleCrashes = true;
```
* `force force`

```
force ace_medical_treatment_clearTrauma = 1;
```
* `force force`
* set to `0` to retain current behavior

```
force ace_medical_treatment_cprSuccessChanceMax = 0.4;
force ace_medical_treatment_cprSuccessChanceMin = 0.4;
```
* both `force force`
* set both to `0.75` to retain current behavior

***
```
force ace_nametags_ambientBrightnessAffectViewDist = 1;
```
* consider `force force` for consistent experience, though it might be cool for some missions if this could be altered

***
```
force ace_overheating_cookoffCoef = 1;
force ace_overheating_coolingCoef = 1;
force ace_overheating_heatCoef = 1;
force ace_overheating_jamChanceCoef = 1;
force ace_overheating_overheatingRateOfFire = true;
force ace_overheating_suppressorCoef = 1;
force ace_overheating_unJamOnSwapBarrel = false;
```
* all of these to `force force`, for consistency with other cookoff options and across missions
* set `ace_overheating_cookoffCoef` to `0` to disable automatic small arms discharge on overheat, preserving current behavior

***
```
force ace_parachute_failureChance = 0;
```
* consider `force force` for consistency

***
```
force ace_ui_enableSpeedIndicator = true;
```
* either `force force` this or remove the `force`, it's a fairly useless triple arrow underneath the soldier stance icon that indicates jogging/sprinting; likely safe to leave player-customizable, or (at least) it doesn't make sense to be mission-customizable

***
### Setting values to consider changing if quick vote is achieved:

```
ace_repair_locationsBoostTraining
```
* to `true` (presumably same as with boosting medical near medical stations)

```
ace_medical_treatment_advancedDiagnose
```
* consider using `2` to allow players reliably diagnosing death / cardiac arrest via the ACE medical system

**Comment by JamesTheClarke** [comment]
I've created three new issues as a reminder that some of the more impactful new features / overhauls require testing:

- #58 
- #59 
- #60

**Comment by JamesTheClarke** [comment]
Unfortunately due to the amount of yesterday's Monthly Meeting agenda topics there was no room for this very lengthy topic, thus I decided to post-pone it until the next Monthly Meeting beginning of February. However, since @freghar 's [test mod for a medical system adaptation requires us updating to the latest version of ACE3](https://www.carpenoctem.co/forums/m/26081621/viewthread/33616530-ace-medical-restoring-hitpoints-on-stitching/post/139911926#p139911926) this topic has become a bit more urgent, as we'd like to start testing already during the hiatus.

Therefore I propose the SSgts and Staff go through the entire proposal summarised in the OP and sign-off on it / give feedback via GitHub rather than originally intended via a Monthly Meeting. Deadline for this would be December 17th 2021 so we can roll it out when the hiatus starts.

Important to note: this first implementation of the latest ACE3 expansion attempts to preserve the current status quo as much as possible, thus Advanced Vehicle Damage, Weapon Overheating/Cookoff and ACE Fire are disabled. Other settings have been set in line with our existing presets for similar settings.

**Review by redteddy — DISMISSED** [review]
I approve of retaining our current gameplay with the settings for the latest version.

File: cba_settings_userconfig/cba_settings.sqf
Comment (freghar) [inline cba_settings_userconfig/cba_settings.sqf:108]: Since this is a new feature, my suggestion was to either set `ace_fire_dropWeapon = 0` or `ace_fire_enabled = false`, neither was implemented.

File: cba_settings_userconfig/cba_settings.sqf
Comment (freghar) [inline cba_settings_userconfig/cba_settings.sqf:116]: Shouldn't this be just a change of default, without `force force`? .. So that non-eng don't get the notification, but people can change this setting anyway?

File: cba_settings_userconfig/cba_settings.sqf
Comment (freghar) [inline cba_settings_userconfig/cba_settings.sqf:310]: Again, this seems like an unnecessary regression to what was previously player-settable.

**Comment by Shadisica** [comment]
Yeah, I'm not seeing any issues here.

File: cba_settings_userconfig/cba_settings.sqf
Comment (freghar) [inline cba_settings_userconfig/cba_settings.sqf:333]: Same here. I would actually be against this change personally, I have it intentionally set to `false`, so repacking doesn't empty my weapon, so I'm always ready to shoot.

**Review by freghar — CHANGES_REQUESTED** [review]
I've provided some comments, see what you think.

File: cba_settings_userconfig/cba_settings.sqf
Comment (JamesTheClarke) [reply cba_settings_userconfig/cba_settings.sqf:108]: Good eye, that was a copy paste on my part in the commit. In the OP I suggested:

// ACE Fire
force force ace_fire_dropWeapon = 1;
force force ace_fire_enabled = false;

Going to fix in the next commit.

**Comment by JamesTheClarke** [comment]
Included Freghar's feedback in the latest commit and updated the OP accordingly.

**Review by redteddy — DISMISSED** [review]

File: cba_settings_userconfig/cba_settings.sqf
Comment (Shiny-CNTO) [inline cba_settings_userconfig/cba_settings.sqf:425]: What are 0 1 and 2 in this case? Is 2 true?

File: cba_settings_userconfig/cba_settings.sqf
Comment (Shiny-CNTO) [inline cba_settings_userconfig/cba_settings.sqf:327]: This is not the standard vanilla toolkit. Is there a reason why we go to a new toolkit? We have to change the engineers in our factions to have the right toolkit this way right?

Comment (Seb105) [reply cba_settings_userconfig/cba_settings.sqf:327]: In previous versions of Arma, the "Toolkit" item was the only Toolkit, CDLC like SOG have added new Toolkit reskins so ACE have changed how the toolkit requirement is handled in order to be compatible with them. "Toolkit" is included within "ace_repair_anyToolKit"

File: cba_settings_userconfig/cba_settings.sqf
Comment (Seb105) [inline cba_settings_userconfig/cba_settings.sqf:154]:
```suggestion
force ace_interaction_enableGroupRenaming = true;
```
This is what allows a group leader to rename their group, this is something we could make use of for when we mess with groups mid mission. Only the group leader can do it, it's not abusable if some does mess with it then they will be easily spotted

File: cba_settings_userconfig/cba_settings.sqf
Comment (Seb105) [inline cba_settings_userconfig/cba_settings.sqf:113]:
```suggestion
force ace_fortify_markObjectsOnMap = 1;
```
Might be some circumstance you'd want this to be customisable.

File: cba_settings_userconfig/cba_settings.sqf
Comment (Shiny-CNTO) [reply cba_settings_userconfig/cba_settings.sqf:327]: Ok clear! Thanks for clarifying this.

**Comment by JamesTheClarke** [comment]
Added suggestions by @Seb105 , pending final review by @Shiny-CNTO and @redteddy we can roll this out.

Updating OP accordingly.

**Comment by Shiny-CNTO** [comment]
Waiting for answers on my questions then I will review again @JamesTheClarke

**Comment by JamesTheClarke** [comment]
> 

I have already answered it. It's a new thrid option that didn't exist before. thus 2 is neither "false" nor "true" as it used to be.

> 2 is the new third option, which gives players the ability tell if a player is dead via ACE3 interaction instead of having to rely on meta tactics such as "can they still be dragged?" etc.

If you wanna know the current definition of true or false open your game and check the addon options.

**Comment by Shiny-CNTO** [comment]
@JamesTheClarke please answer questions on conversation in the conversation and not on the entire pull request. 
Very hard to follow if we all just reply to the entire PR instead of individual topics.

File: cba_settings_userconfig/cba_settings.sqf
Comment (Shiny-CNTO) [reply cba_settings_userconfig/cba_settings.sqf:425]: Advanced Diagnose: Checkbox to enable “Check Pulse”, “Check Blood Pressure” and “Check Response” treatment actions, if disabled CPR will only be shown when performing CPR is necessary. This setting is imperative to determining if a patient is unconscious or in cardiac arrest.

How is this a checkbox if it has 0, 1 and 2 as a value? I have to go in game and actually check them out.

Comment (Shiny-CNTO) [reply cba_settings_userconfig/cba_settings.sqf:425]: image
These are the options that reflect 0,1 and 2. 2 sounds the best so I agree :)

**Review by Shiny-CNTO — APPROVED** [review]
Approved. Left comments in conversations on why and how.

**Review by redteddy — APPROVED** [review]

**Pull request merged**
